packages/client/components/AvatarList.tsx

```diff
@@ -56,11 +67,14 @@ const AvatarList = (props: Props) => {
       {transitionChildren.length === 0 && emptyEl}
       {transitionChildren.map(({onTransitionEnd, child, status, displayIdx}) => {
         const {id: userId} = child
+        const transitionSpeed =
```

Comment:
This feels a bit fiddly but without it, the meeting card shrinks a bit too slowly when the AvatarList is removed, i.e. when a connected user leaves the meeting

Reply:
Thanks, I really appreciate getting detailed explanations like this where you go through your thought pattern!